pkg/server/server.go

```diff
@@ -122,7 +123,12 @@ func (s *Server) run(ctx context.Context) (err error) {
 		return err
 	}
 
-	endpointsServer, err := s.newEndpointsServer(ctx, cat, svidRotator, serverCA, metrics, caManager)
+	policyEngine, err := policy.NewEngineOrDefault(ctx, s.config.PolicyEngineConfig)
+	if err != nil {
+		return err
```

Comment:
We should format this error to provide an indication of the context, otherwise it may be difficult to guess that the error happened while creating a new authorization policy engine.

Reply:
fixed